cache/redis_client.go

```diff
@@ -185,10 +193,28 @@ func NewRedisClient(logger log.Logger, name string, config RedisClientConfig, re
 	}
 	if config.MaxGetMultiConcurrency > 0 {
 		c.getMultiGate = gate.New(
-			prometheus.WrapRegistererWithPrefix("redis_getmulti_", reg),
+			prometheus.WrapRegistererWithPrefix(getMultiPrefix, reg),
 			config.MaxGetMultiConcurrency,
 		)
 	}
+
+	c.clientInfo = promauto.With(reg).NewGaugeFunc(prometheus.GaugeOpts{
+		Name: "redis_client_info",
```

Comment:
This metric name is not consistent with the memcached client.

Reply:
As mentioned in https://github.com/grafana/dskit/pull/255#discussion_r1098161871 memcached will produce `<prefix>_memcached_client_info` as this redis format. The reason to keep cache name in the metric name because different cache will expose different label.

Same question do you think we still need to remove cache from metric name and move it to `backend` label with the above reason?

Reply:
What you did is correct. I would just add a comment both here and in the memcached one to explain why the metric names here are specific.